inflation/main.py

```diff
@@ -1,29 +1,40 @@
 from datetime import datetime
 import logging
 import os
+import gzip
+import json
 
 from fastapi import BackgroundTasks, FastAPI
 
 from common.clients.google_storage_client import GoogleStorageClient
 from common.customized_logging import configure_logging
 from inflation.config import settings
-from inflation.dataset.crawl import A101Crawler, CrawlerManager, MigrosCrawler
+from inflation.dataset.crawl import CrawlerManager, A101Crawler, MigrosCrawler
+from inflation.dataset.parse import ParserManager, A101Parser, MigrosParser
 
-BUCKET_NAME = os.getenv("GCS_BUCKET", "inflation-in-turkey")
+
+CRAWLER_BUCKET = os.getenv("CRAWLER_BUCKET", "inflation-project-crawler-output")
+PARSER_BUCKET = os.getenv("PARSER_BUCKET", "inflation-project-parser-output")
 
 CRAWLERS = {"a101": A101Crawler(), "migros": MigrosCrawler()}
+PARSERS = {"a101": A101Parser(), "migros": MigrosParser()}
 
 configure_logging()
 logger = logging.getLogger(__name__)
 
 app = FastAPI()
 cm = CrawlerManager(CRAWLERS)
-storage_client = GoogleStorageClient(bucket_name=BUCKET_NAME)
+pm = ParserManager(PARSERS)
+crawler_client = GoogleStorageClient(bucket_name=CRAWLER_BUCKET)
+parser_client = GoogleStorageClient(bucket_name=PARSER_BUCKET)
```

Comment:
Let's move this to managers actually. We don't really need to know this here. Manager can handle. This is the next PR, though. don't address here but take note.

Reply:
ok